src/Tribe/Cache.php

```diff
@@ -100,7 +102,7 @@ public function set_transient( $id, $value, $expiration = 0, $expiration_trigger
 	 * @return mixed
 	 */
 	public function get( $id, $expiration_trigger = '', $default = false, $expiration = 0, $args = array() ) {
-		$group = in_array( $id, $this->non_persistent_keys ) ? 'tribe-events-non-persistent' : 'tribe-events';
+		$group = in_array( $id, $this->non_persistent_keys ) ? 'tribe-events-non-persistent' : self::PERSISTENT_GROUP;
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 118 characters

<sub>hash: 249f054ff4b5a5edc7b27f7cc8e5cc2c</sub>